### PR TITLE
Add landing message #1

### PR DESCRIPTION
**Pull request by hafiyuddin**
This is the updated code for landing page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
-    
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (HafiyAzhad) [inline index.html]: Could you remove the "Enjoy your stay." string and make the "Welcome to PotionFy!" string red?

Comment (hafiyuddin) [reply index.html]: Sure, can do!

**Review by HafiyAzhad — CHANGES_REQUESTED** [review]
please, change the code as per my comment on the review

**Review by HafiyAzhad — APPROVED** [review]
Well done Harry!

**Pull request merged**